.github/workflows/dart_package.yml

```
@@ -27,6 +27,10 @@ on:
         required: false
         type: string
         default: "vm"
+      check_ignore:
+        required: false
+        type: boolean
+        default: true
```

```suggestion
        default: false
```

This is a breaking change as we retag constantly on `v1` so to ensure we dont change the potential output of anyone on the v1 tag we need to make this false